dobson/main.py

```diff
@@ -86,6 +108,30 @@ def respond_users(slack_client, channel):
     )
 
 
+def list_to_str(lst):
+    """
+    Returns a comma separated string for the passed list
+    >>> list_to_str(['sean', 'jason', 'nikhil'])
+    'sean, jason and nikhil'
+    >>> list_to_str(['sean', 'jason'])
+    'sean and jason'
+    >>> list_to_str(['sean'])
+    'sean'
+    >>> list_to_str([])
+    ''
+    """
+    if len(lst) > 2:
+        res = '{}, and {}'.format(', '.join(lst[:-1]), lst[-1])
+    elif len(lst) == 2:
+        res = '{} and {}'.format(lst[0], lst[1])
+    elif len(lst) == 1:
+        res = '{}'.format(lst[0])
+    else:
+        res = ''
+
+    return res
```

Comment:
I'd just return at each statement here since nothing happens with the string after it's made, so like replace all the `res = ` with just `return`.